generic_config_updater/services_validator.py

```diff
@@ -71,3 +71,35 @@ def vlan_validator(old_config, upd_config, keys):
     # No update to DHCP servers.
     return True
 
+def caclrule_validator(old_config, upd_config, keys):
+    old_acltable = old_config.get("ACL_TABLE", {})
+    upd_acltable = upd_config.get("ACL_TABLE", {})
+
+    try:
+        old_cacltable = [table for table in old_acltable
```

Comment:
>old_acltable [](http://example.com/codeflow?start=44&length=12)

You can iterate `old_acltable.items()` and get key and value at the same time. It will simplify the code.